src/index.css

```diff
@@ -128,3 +129,68 @@ a {
   grid-row: 2;
   grid-column-end: 1;
 }
+
+.circular-progress-container {
+  width: 5.625rem;
+  height: 5.625rem;
+  padding: 0.63rem 0.625rem 0.75rem 0.75rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.circular-progress {
+  position: absolute;
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #307bbe;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.progress-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.percent-complete {
+  font-size: 1.5rem;
+}
+
+.progress-detail {
+  margin: 0;
+}
+
+.progress-divider {
+  width: 0.125rem;
+  height: 5.375rem;
+  margin: 20px 3.688rem 0 2.938rem;
+  background-color: #e8e8e8;
+}
+
+.book-genre {
+  font-weight: 500;
+  color: #525252;
+}
+
+.book-author {
+  font-size: 0.875rem;
+  font-weight: 400;
+  color: #4386bf;
+  margin-bottom: 2rem;
+}
```

Comment:
- [ ] Great job with your design 👏 . Kindly make sure your text is aligned on the same line as the rest of the text as seen in the provided design 🙏 
- [ ] Kindly make sure you use the correct fonts as seen. You declared `roboto slab` but didn't import it. Kindly fix this :pray:

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/201176364-fa281d71-6577-4498-854a-379a2158b055.png)  |  ![image](https://user-images.githubusercontent.com/74814780/201176333-aba64de1-15a3-4dbe-b813-95962c51b77b.png)